src/modules/dashboard/transformers/transformers.ts

```diff
@@ -253,9 +254,29 @@ export const getActionsListData = (
                 },
                 name,
               } = unformattedAction;
-              const actionEvent = Object.entries(ACTIONS_EVENTS).find((el) =>
-                el[1]?.includes(name.split('(')[0]),
-              );
+
+              let actionEvent;
+
+              if (
+                name.split('(')[0] ===
```

Comment:
I think we have helper for it (formatEventName)

Reply:
Thank you! I didn't knew that